kolibri/core/analytics/pskolibri/_psosx.py

```diff
@@ -0,0 +1,3 @@
+"""macOS platform implementation."""
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
Seems empty :)

Reply:
It is. OSX version is not started, not even sure if it is worth doing it.

Reply:
After chatting with @indirectlylit , we are not going to do OSX by now